packages/desktop-client/src/components/reports/graphs/showActivity.ts

```diff
@@ -33,22 +37,32 @@ export function showActivity({
   endDate,
   field,
   id,
+  interval = 'Day',
 }: showActivityProps) {
   const amount =
```

Comment:
This probably needs renamed to better reflect the purpose of the variable.  Something like `isOutflow` might be better.